### PR TITLE
Add feature branch

### PR DESCRIPTION
**Pull request by mr-tms**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,71 +1,12 @@
...
-  }
-});
-
-server.listen(8000, '127.0.0.1', () => {
-  console.log('Listening to request on port 8000');
-});
+require('dotenv').config();
+const express = require('express');
+const app = express();
+const port = 8000;
```
Comment (LiudasDEM) [inline index.js]: could also be set as env variable

File: .env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+DB_HOST=localhost
```
Comment (LiudasDEM) [inline .env]: never check-in .env files

File: index.js [hunk 3]
```diff
@@ -1,71 +1,12 @@
...
+
+app.get('/', (req, res) => {
+  res.send('Hello World')
+})
+
+app.listen(port, () => {
+  console.log(`App listening at port ${port}`);
+})
```
Comment (LiudasDEM) [inline index.js]: configure .editorconfig to add newline at eof

File: index.js [hunk 4]
```diff
@@ -1,71 +1,12 @@
...
+const express = require('express');
+const app = express();
+const port = 8000;
+
+app.get('/', (req, res) => {
+  res.send('Hello World')
+})
+
+app.listen(port, () => {
+  console.log(`App listening at port ${port}`);
```
Comment (LiudasDEM) [inline index.js]: don't use console.log, only allow warn/error/info/group and similar. configure eslint rule for that

File: package.json [hunk 5]
```diff
@@ -1,18 +1,32 @@
...
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1"
   },
   "devDependencies": {
-    "nodemon": "^2.0.6"
+    "eslint": "^7.11.0",
+    "eslint-config-airbnb": "^18.2.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "eslint-plugin-react": "^7.21.5",
```
Comment (LiudasDEM) [inline package.json]: are these necessary for backend project?

File: package.json [hunk 6]
```diff
@@ -1,18 +1,32 @@
...
   "scripts": {
     "start": "nodemon index.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mr-tms/nodejs.git"
+  },
+  "author": "mr-tms",
```
Comment (LiudasDEM) [inline package.json]: good practice is to add your email as well. e.g. "author": "mr-tms <marijus.tumosa@gmail.com>"

File: .eslintrc.json [hunk 7]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (LiudasDEM) [inline .eslintrc.json]: I strongly rekomend to use javascript instead of json for babel/eslint and similar configuration files

Comment (mr-tms) [reply .eslintrc.json]: Changed to js. But could you elaborate why is it recommended?

File: .eslintrc.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+    'airbnb-base',
+  ],
+  parserOptions: {
+    ecmaVersion: 12,
+  },
+  rules: {
+    'semi': ['error', 'always'],
+    'quotes': ['error', 'single'],
+    'eol-last': ['error', 'always'],
+    'no-console': ['error', { 'allow': ['warn', 'error', 'info', 'group'] }]
```
Comment (LiudasDEM) [inline .eslintrc.js]: missing dangling comma. also configure eslint for it

File: package.json [hunk 9]
```diff
@@ -1,18 +1,29 @@
...
   "scripts": {
     "start": "nodemon index.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "author": "",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mr-tms/nodejs.git"
+  },
+  "author": "mr-tms marijus@tumosa.lt",
```
Comment (LiudasDEM) [inline package.json]: email should be wrapped into < >

**Review by LiudasDEM — APPROVED** [review]

**Pull request merged**